src/validations/test/rules/.gitignore

```diff
@@ -0,0 +1 @@
+*.html
```

Comment:
There are other ignore patterns in the root `.gitignore` - could we either move this one there, or add the other patterns to a new one?

Reply:
Good catch. I had though to add in the spot where these occurred but the root is better.
The root, btw, missed these XSpec HTML artifacts because the hierarchy changed. I'll correct those.